demo/full/styles/style.css

```diff
@@ -3,18 +3,18 @@ body {
 }
 
 @font-face {
-  font-family: 'icons';
+  font-family: "icons";
   src: url("../fonts/FontAwesome.otf");
   font-weight: normal;
   font-style: normal;
 }
 
 .icon {
-  font-family: "icons";
+  font-family: "icons", sans-serif;
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Unexpected missing generic font family ([css:S4649](https://sonarcloud.io/organizations/rx-player/rules#rule_key=css%3AS4649))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=264&issues=AWTMwmjYi6fIhhPMqNRS&open=AWTMwmjYi6fIhhPMqNRS)

[ISSUE_KEY]: AWTMwmjYi6fIhhPMqNRS